src/logger.ts

```diff
@@ -0,0 +1,2 @@
+import consola from 'consola'
+export default consola.withScope('nuxt:yandex-metrika')
```

Comment:
we now have a handy new utility in kit which means we can also remove the consola dependency.
```suggestion
import { useLogger } from '@nuxt/kit'
export default useLogger('nuxt:yandex-metrika')
```